### ISSUE TITLE
AttributeError: 'DisplayString' object has no attribute '_value'

### ISSUE THREAD

**Issue by romerojunior** [etingof/pysnmp#95]
Running pysnmp __version__ = '4.3.10' (installed via pip) on a clean CentOS Linux release 7.4.1708 (Core) intallation (Python 2.7.5) and calling SnmpEngine() immediately returns the following traceback:

```
>>> from pysnmp.hlapi import *
>>> SnmpEngine()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python2.7/site-packages/pysnmp/entity/engine.py", line 63, in __init__
    self.msgAndPduDsp = MsgAndPduDispatcher()
  File "/usr/lib/python2.7/site-packages/pysnmp/proto/rfc3412.py", line 32, in __init__
    'SNMP-TARGET-MIB', 'SNMP-USER-BASED-SM-MIB'
  File "/usr/lib/python2.7/site-packages/pysnmp/smi/builder.py", line 362, in loadModules
    self.loadModule(modName, **userCtx)
  File "/usr/lib/python2.7/site-packages/pysnmp/smi/builder.py", line 330, in loadModule
    'MIB module \"%s\" load error: %s' % (modPath, traceback.format_exception(*sys.exc_info()))
pysnmp.smi.error.MibLoadError: MIB module "/usr/lib/python2.7/site-packages/pysnmp/smi/mibs/SNMPv2-MIB.pyc" load error: [
'Traceback (most recent call last):\n', 
'  File "/usr/lib/python2.7/site-packages/pysnmp/smi/builder.py", line 325, in loadModule\n    
exec (modData, g)\n', 
'  File "/tmp/pip-build-NoLK36/pysnmp/pysnmp/smi/mibs/SNMPv2-MIB.py", line 27, in <module>\n', 
'  File "/usr/lib/python2.7/site-packages/pysnmp/proto/rfc1902.py", line 205, in subtype\n   
 return univ.OctetString.subtype(self, *args, **kwargs).setFixedLength(self.getFixedLength())\n', 
'  File "/usr/lib/python2.7/site-packages/pyasn1/type/base.py", line 122, in subtype\n
    value = self._value\n', 
"AttributeError: 'DisplayString' object has no attribute '_value'\n"]
```

Haven't found any reference for this error. Any suggestions?

**Comment by etingof** [comment]
Thanks for your report!

I suspect you are using some obsolete pyasn1 version. Can you show me your `pip list` please?

So far I can't reproduce this problem, here is how I test it:

```
$ virtualenv-2.7 xxx
New python executable in /tmp/xxx/bin/python2
Also creating executable in /tmp/xxx/bin/python
Installing setuptools, pip, wheel...done.
[ietingof@garlic tmp]$ . xxx/bin/activate
(xxx) [ietingof@garlic tmp]$ pip install pysnmp
Collecting pysnmp
  Downloading pysnmp-4.3.10-py2.py3-none-any.whl (283kB)
    100% |████████████████████████████████| 286kB 902kB/s 
Collecting pysmi (from pysnmp)
  Downloading pysmi-0.1.4-py2.py3-none-any.whl (76kB)
    100% |████████████████████████████████| 81kB 1.2MB/s 
Collecting pyasn1>=0.2.3 (from pysnmp)
  Downloading pyasn1-0.3.7-py2.py3-none-any.whl (63kB)
    100% |████████████████████████████████| 71kB 1.2MB/s 
Collecting pycryptodome (from pysnmp)
Collecting ply (from pysmi->pysnmp)
Installing collected packages: ply, pysmi, pyasn1, pycryptodome, pysnmp
Successfully installed ply-3.10 pyasn1-0.3.7 pycryptodome-3.4.7 pysmi-0.1.4 pysnmp-4.3.10
$ python
Python 2.7.13 (default, May 10 2017, 20:04:28) 
[GCC 6.3.1 20161221 (Red Hat 6.3.1-1)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from pysnmp.hlapi import *
>>> SnmpEngine()
SnmpEngine(snmpEngineID=SnmpEngineID())
>>> 
```

**Comment by romerojunior** [comment]
```
pyasn1 (0.2.3)
pysnmp (4.3.10)
pysnmp-apps (0.4.2)
pysnmp-mibs (0.1.6)
```

**Comment by romerojunior** [comment]
Issue solved after updating `pyasn1-0.2.3` to `pyasn1-0.3.7`.
Thanks for the tip, @etingof!

**Comment by etingof** [comment]
Somehow I still can't reproduce this:

```
$ python
Python 2.7.10 (default, Jul 30 2016, 19:40:32) 
[GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.34)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from pysnmp.hlapi import *
>>> SnmpEngine()
SnmpEngine(snmpEngineID=SnmpEngineID())
>>> import pyasn1
>>> pyasn1.__version__
'0.2.3'
>>> import pysnmp
>>> pysnmp.__version__
'4.4.1'
>>> 
```

Hope this is just a misconfiguration on your side.
